pylint_odoo/checkers/no_modules.py

```diff
@@ -389,6 +396,11 @@ def visit_call(self, node):
             args = misc.join_node_args_kwargs(node)
             for argument in args:
                 argument_aux = argument
+                # Check this 'param = fields.Char("param")'
+                if (isinstance(argument, astroid.Const) and
+                              (argument.value.lower().replace(' ', '_') ==
+                               argument.parent.parent.targets[0].name)):
+                    self.add_message('attribute-string-redundant', node=node)
```

Comment:
Should we validate just the first argument for the cases where is used as first argument?
I mean, this code is validating all arguments
For the case of `fields.Char(arg1, arg2, arg3, arg4)` we are validating 4 times the same pattern instead of validating just for the first one